src/components/AlertMessage/AlertMessage.tsx

```diff
@@ -22,7 +21,6 @@ const useStyles = makeStyles((theme: Theme) => ({
 
 export default function AlertMsg({ severity = "error", msg }: IAlertMsg) {
 	const classes = useStyles();
-	const theme = useTheme();
 	return (
 		<div className={classes.root}>
 			<Alert severity={severity} className={classes.alertmsg}>
```

Comment:
I dont think there is any use of creating a Wrapper component around the `Alert` component provided by material UI.  
It can be used directly